src/main/java/ca/warp7/frc2023/Constants.java

```diff
@@ -1,3 +1,8 @@
 package ca.warp7.frc2023;
 
-public class Constants {}
+public class Constants {
+
+    // fourbar constants
+    public static final int FOURBAR_MOTOR_PORT_0 = 0;
```

Comment:
Oops I missed this in the review... The naming convention that we use for comments is lowercase k then camel case of the rest of the name. E.g. kFourbarMotorID